packages/stentor-response/src/ResponseBuilder.ts

```diff
@@ -106,10 +123,38 @@ export class ResponseBuilder<T = Response<ResponseOutput>> extends AbstractRespo
         return this;
     }
 
+    /**
+     * Provides active context that the NLU can use to help select the next intent.
+     */
+    public withActiveContext(context: ActiveContext | ActiveContext[]): ResponseBuilder<T> {
+
+        if (!context) {
+            return this;
+        }
+
+        let active: ActiveContext[] = this._response?.context?.active || [];
+
+        if (Array.isArray(context)) {
+            active = active.concat(context);
```

Comment:
`active = active.concat(context);` would just work.  concat takes and array or individual

Reply:
I was wondering about that, will update.  Thanks